x-pack/plugins/maps/public/classes/sources/es_geo_grid_source/es_geo_grid_source.js

```diff
@@ -229,27 +265,21 @@ export class ESGeoGridSource extends AbstractESAggSource {
     return features;
   }
 
-  // Do not use composite aggregation when there are no terms sub-aggregations
-  // see https://github.com/elastic/kibana/pull/57875#issuecomment-590515482 for explanation on using separate code paths
-  async _nonCompositeAggRequest({
+  _addNonCompositeAggregationsToSearchSource(
     searchSource,
-    indexPattern,
-    precision,
-    layerName,
-    registerCancelCallback,
-    bufferedExtent,
-  }) {
+    { indexPattern, precision, bufferedExtent }
```

Comment:
Why are indexPattern, precision, bufferedExtent put into an object? Could you instead just pass ithem in as separate parameters?